conformance/ion_encoding/load_symtab.ion

```diff
@@ -0,0 +1,77 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+// Exact-match imports
+
+(ion_1_1
+  (toplevel $ion_encoding::((load abc "abcs" 1)
+                            (symbol_table abc)))
+  (then (toplevel '#$12')
+        (produces a))
+  (then (toplevel '#$13')
+        (signals "Invalid symbol address:")))
+
+(ion_1_1
+  (toplevel $ion_encoding::((load mno "mnop" 3)
+                            (load abc "abcs" 1)
+                            (symbol_table abc mno)))
+  (then (toplevel '#$12' '#$13' '#$15')
+        (produces a m o))
+  (then (toplevel '#$16')
+        (signals "Invalid symbol address:")))
+
+
+// Absent shared symbol
+
+(ion_1_1
+  (toplevel $ion_encoding::((load mno "mnop" 4)
+                            (symbol_table mno)))
+  (then (toplevel '#$12' '#$13')
+        (denotes (Symbol (absent "mnop" 1)) (Symbol "n"))))
+
+
+// Module name shadowing
+
+(ion_1_1
```

Comment:
Can you add a plain english description of the expected behavior? Would this be accurate? "When two modules are imported with the same name, the later one supersedes the first."